hook-janitor/src/webhooks.rs

```diff
@@ -198,10 +198,10 @@ impl WebhookCleaner {
     async fn get_completed_rows(&self, tx: &mut SerializableTxn<'_>) -> Result<Vec<CompletedRow>> {
         let base_query = format!(
             r#"
-            SELECT DATE_TRUNC('hour', finished_at) AS hour,
-                (metadata->>'team_id')::bigint AS team_id,
-                (metadata->>'plugin_config_id')::bigint AS plugin_config_id,
-                count(*) as successes
+            SELECT DATE_TRUNC('hour', last_attempt_finished_at) AS hour,
+                   (metadata->>'team_id')::bigint AS team_id,
+                   (metadata->>'plugin_config_id')::bigint AS plugin_config_id,
+                   count(*) as successes
             FROM {0}
             WHERE status = 'completed'
               AND queue = $1
```

Comment:
Just for consistent indenting? Seems like it's 4 spaces everywhere, but we added an extra 3 here:
```suggestion
                (metadata->>'team_id')::bigint AS team_id,
                (metadata->>'plugin_config_id')::bigint AS plugin_config_id,
                count(*) as successes
            FROM {0}
            WHERE status = 'completed'
                AND queue = $1
```